plugins/redfish/fu-redfish.rs

```diff
@@ -17,6 +17,45 @@ struct RedfishProtocolOverIp {
     service_hostname_len: u8,
 // optional service_hostname goes here
 }
+
+#[derive(ToString)]
+enum RedfishInterfaceType {
+    UsbNetwork = 0x02,
+    PciNetwork = 0x03,
+    UsbNetworkV2 = 0x04,
+    PciNetworkV2 = 0x05,
+}
+
+enum RedfishIpAssignmentType {
+    Static = 0x00,
+    Dhcp = 0x02,
+    AutoConfig = 0x03,
+    HostSelect = 0x04,
+}
+
+enum RedfishIpAddressFormat {
+    Unknown = 0x00,
+    V4 = 0x01,
+    V6 = 0x02,
+}
+
+#[repr(u8)]
+enum RedfishControllerInterfaceType {
+    NetworkHost = 0x40,
+}
```

Comment:
isn't this a bit heavy for a single value?  Or are you thinking you'll eventually have more?

Reply:
Yup, they'll be more in the future.